lib/core/schedule.ts

```diff
@@ -0,0 +1,19 @@
+export type Schedule = {
+  id: string;
+  description?: string;
+  cron: string;
+  tags?: Record<string, string>;
+  promiseId: string;
+  promiseTimeout: number;
+  promiseParam?: PromiseValue;
+  promiseTags?: Record<string, string>;
+  lastRunTime?: number;
+  nextRunTime?: number;
+  idempotencyKey?: string;
+  createdOn?: number;
+};
+
+export type PromiseValue = {
```

Comment:
We could just embed this type rather than name it, that's how we currently do it in promise.ts

Reply:
Fixed.